share/doc/wake/tutorial.md

```diff
@@ -560,3 +560,129 @@ As a final note, wake files must be UTF-8 encoded.
 This means that you can use Unicode symbols in strings, comments,
 identifiers, and even operators.
 In fact, all the operators we just printed are legal in wake.
+
+## Dealing with failure
+
+Previously, we glossed over functions like `getOrElse` and `getWhenFail`.
+In this section we will explain those as well as the more general concept
+of dealing with operations that can fail in a build flow.
+
+Many operations can fail like taking the first element (`head`) of a `List`
```

Comment:
Is there a different word than "fail" that we can use here? I'm a bit worried that the word "fail" may be conflated with the `Fail` from `Result` rather than the more colloquial "a function may not perform desired action". I think people who understand FP will "get it", but people who are totally new may have more trouble grasping why `Option`s are useful and that we have both `Option` and `Result` in `wake`.

Even if it sounds awkward, I think even "unsuccessful" or "not succeed" may be a better word to use here because there's no name conflict with a capital F Fail.

Reply:
I'm open to this, but saying stuff like 
"Many operations may not succeed like taking the first element (`head`) of a `List`" seems less clear to me. I agree with the idea of avoiding the name aliasing though